### DISCUSSION TITLE
Permission variable not showing up in dashboard permission dropdown select

### DISCUSSION THREAD

**Discussion by felixhagspiel** [nhost/nhost#2386] [Q&A] [answered]
hey all,
I am currently trying to setup a permissions variable for my `profile` table as mentioned here: https://docs.nhost.io/graphql/permissions#permission-variables

Here is what I have:

1. I got a profile table that has a column userId, which has a relation to auth.user.id
2. I have added a new permission variable `X-Hasura-profile-id`
3. I pushed everything, ran `nhost config pull` and `nhost up` in my local environment

I checked my users JWT token and it has `x-hasura-profile-id` included in the claims with the correct profile ID. However, I cannot select this in the dropdown when I am trying to setup a new rule, it just does not show up:
<img width="926" alt="Bildschirmfoto 2023-11-26 um 23 27 29" src="https://github.com/nhost/nhost/assets/5927172/f6946823-a845-4b2a-905e-f7e7e43dabf6">

Am I missing something? When I copy & paste `X-Hasura-profile-id` it just gets overwritten with `X-Hasura-user-id`

**Comment by felixhagspiel** [comment]
for now I found a workaround. I just select `X-Hasura-user-id` in the dropdown, save, and then in the yaml file for the table I manually change it to:

```
filter:
        creatorProfileId:
          _eq: X-Hasura-profile-id
```
which works when you run `nhost up` afterwards. The permission shows up in the dashboard, but only for the changed field, I still cannot select it on any other table. So I guess the only issue is that the permission dropdown select does not know about the custom claims.

**Comment by dbarrosop** [comment, marked as answer]
Thanks for reporting, this seems to be a bug so I created an issue.

https://github.com/nhost/nhost/issues/2387
